samples: net: http_server: No fatal error for setup_server (#17659)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

Networking samples
------------------

* :ref:`http_server` sample:

>   * Updated:
> 
>     * Do not fail with a fatal error if IPv4 or IPv6 server setup fails.

Reviewer: ```suggestion
  * Fixed not to fail with a fatal error if IPv4 or IPv6 server setup fails.
```
